f2/src/pdf/BusinessInfoView.js

```diff
@@ -39,7 +61,7 @@ export const BusinessInfoView = (props) => {
           />
           <DescriptionItemView
             title="confirmationPage.businessInfo.numberOfEmployee"
-            description={businessInfo.numberOfEmployee}
+            description={numberofEmployeeString}
```

Comment:
Change Line64 to "description={<Trans id={businessInfo.numberOfEmployee} />}" and it should work. Don't need code from line 18 -39

Reply:
Just tried  "description={lang[businessInfo.numberOfEmployee]}", and creating PDF crushed.   

Since we need to dig into  deeper layer of "lang['analystReport.numberOfEmployee.1To99'] etc, and keep consistent with what is displayed on summary and analyst report, this is why line 18-39 is used for.

Reply:
I just tried it on my local, it works

Reply:
Please refer to PR 2434, it seems working well
![numberOfEmployee](https://user-images.githubusercontent.com/68352206/97488104-2aae6c80-1934-11eb-8f12-4208bae7435a.png)

Reply:
Tried again, it worked, all pushed